server.py

```diff
@@ -0,0 +1,19 @@
+from win32event import CreateMutex
```

Comment:
Where does this module come from? (and will requiring it impact the ability to create a mac os installer?)